evaluator/src/main/kotlin/RuleSet.kt

```diff
@@ -66,7 +66,11 @@ class RuleSet(val ortResult: OrtResult) {
             scope: Scope,
             project: Project
         ) {
-            val pkg = ortResult.analyzer?.result?.packages?.find { it.pkg.id == pkgRef.id }?.pkg!!
+            val pkg = ortResult.analyzer?.result?.packages?.find { it.pkg.id == pkgRef.id }?.pkg
+                ?: ortResult.analyzer?.result?.projects?.find { it.id == pkgRef.id }?.toPackage()
```

Comment:
Slightly less duplication:

            val pkg = ortResult.analyzer?.result?.let { result ->
                result.packages.find { it.pkg.id == pkgRef.id }?.pkg
                    ?: result.projects.find { it.id == pkgRef.id }?.toPackage()
            }

Reply:
Done.